pi-apps-terminal-bash-edition.sh

```diff
@@ -245,7 +324,7 @@ while [ "$1" != "" ]; do
 			esac
 			exit $?
 		;;
-		info)
+		info|details)
```

Comment:
Could you update the `help` command and the readme to add that `details` also works please?

Reply:
Fixed in https://github.com/Itai-Nelken/PiApps-terminal_bash-edition/pull/4/commits/65a56094d26dae8840f3d2b436cf7246bfd05dd8 and https://github.com/Itai-Nelken/PiApps-terminal_bash-edition/pull/4/commits/f08c3fecd1e297776814fc468ee7b830aa37ba46;